Formula/mp.rb

```diff
@@ -0,0 +1,13 @@
+class Mp < Formula
```

Comment:
Use `MinimumProfit` as a formula name (and rename file accordingly: `minimum-profit.rb`)

Reply:
Thanks, done.